Examples/ArcGISToolkitExamples/ARExample.swift

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ARKit
+import ArcGISToolkit
+import ArcGIS
+
+open class ARExample: UIViewController {
```

Comment:
Since this class isn't used outside of this module (the app), it doesn't need to be `open` (or `public`). It would be best to just use the default (`internal`). Same for all of its members.

Reply:
Done.